child_compassion/controllers/web_children_hold.py

```diff
@@ -66,11 +66,13 @@ def hold_children_job(session, research_id):
         active_id=research_id).sudo()
     expiration_date = datetime.now() + timedelta(minutes=15)
 
+    user_id = session.env['res.users'].search([('name', '=', 'Roma Reber')])
```

Comment:
Add `.id` at the end, to get the id instead of ther record. Name should be `Reber Rose-Marie` to find it.